krun/scheduler.py

```diff
@@ -383,7 +383,7 @@ def run(self, mailer, dry_run=False):
             try:
                 self.sched.platform.check_throttle_counts(self.sched.manifest)
                 measurements = util.check_and_parse_execution_results(
-                    stdout, stderr, rc, self.sched.config, instrument=vm_def.instrument)
+                    stdout, stderr, rc, self.sched.config, self.key, instrument=vm_def.instrument)
```

Comment:
I might be missing out on something, but this is only the assignment of `key` to a field -- nothing seems to read from it?

Reply:
It's passing `self.key` as a function argument so that we can print it in the log.

Reply:
I'm an idiot.

Reply:
Don't sweat it. Functions with that many args become hard to scan :P